books.css

```diff
@@ -127,17 +127,17 @@ input {
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 4rem 0 4rem 0;
+  padding-top: 2rem;
 }
 
 .header {
   width: 100%;
+  display: flex;
```

Comment:
- [ ] You have used `display: flex` 9 times, which falls under the category of bad CSS practice since you are repeating yourself a lot. I would recommend you create a class with display: flex on it and use it whenever you need on your HTML